packages/editor/src/components/block-list/style.scss

```diff
@@ -808,10 +808,10 @@
 	}
 
 	// Don't do it for wide elements, this causes a horizontal scrollbar.
-	&[data-align="full"] .editor-block-contextual-toolbar {
+	/*&[data-align="full"] .editor-block-contextual-toolbar {
```

Comment:
Is this meant to be removed?

Reply:
Yes, sorry about that.

That rule was needed at one point, but appears to have been refactored out. There's a lot of cleanup in this PR. I pushed a fix to remove it.